pom.xml

```diff
@@ -267,7 +267,11 @@
         <version>${selenium.version}</version>
       </dependency>
       <!-- == End Test == -->
-
+      <dependency>
```

Comment:
- [ ] You should write that we are specifying a version for workaround.

Reply:
@bttanakaysd 
42d495bc3bc8cdafb898bc4f9d99817e3e2b4a47 
I added comment why we specify version .

Reply:
@bttanakaysd 
949cd29ef8b26fb87b916cd2cf163a7d1d2361e4 
I modified comment to smart .